mapper/related_data.go

```diff
@@ -0,0 +1,49 @@
+package mapper
+
+import (
+	"net/http"
+
+	"github.com/ONSdigital/dp-api-clients-go/v2/zebedee"
+	"github.com/ONSdigital/dp-frontend-search-controller/cache"
+	"github.com/ONSdigital/dp-frontend-search-controller/config"
+	"github.com/ONSdigital/dp-frontend-search-controller/data"
+	"github.com/ONSdigital/dp-frontend-search-controller/model"
+	coreModel "github.com/ONSdigital/dp-renderer/v2/model"
+	searchModels "github.com/ONSdigital/dp-search-api/models"
+	topicModel "github.com/ONSdigital/dp-topic-api/models"
+)
+
+func CreateRelatedDataPage(cfg *config.Config, req *http.Request, basePage coreModel.Page,
+	validatedQueryParams data.SearchURLParams, respC *searchModels.SearchResponse, lang string, homepageResponse zebedee.HomepageContent, errorMessage string,
+	navigationContent *topicModel.Navigation, template string, topic cache.Topic, validationErrs []coreModel.ErrorItem, zebedeeResp zebedee.PageData, bc []zebedee.Breadcrumb,
+) model.SearchPage {
+	page := model.SearchPage{
+		Page: basePage,
+	}
+
+	page.Metadata.Title = "Data related to " + zebedeeResp.Description.Title
```

Comment:
How are we dealing with welsh of this? Does Babbage have the welsh?

Reply:
Babbage possibly does have it but when it comes to titles the dp-renderer expects a string so titles are always hardcoded or pulled from data rather than having a locales value

See:
https://github.com/ONSdigital/dp-frontend-search-controller/blob/develop/mapper/mapper.go#L33
https://github.com/ONSdigital/dp-frontend-filter-dataset-controller/blob/develop/mapper/mapper.go#L56